ui/app/AppLayouts/Profile/stores/ContactsStore.qml

```diff
@@ -43,11 +46,35 @@ QtObject {
     // Support models for showcase for a contact with showcasePublicKey
     readonly property var showcaseCollectiblesModel: d.contactsModuleInst.showcaseCollectiblesModel
 
+    // Adapted contacts showcase models (TODO: Review if the store is the proper place. Should it be in an intermediate
+    // layer in between of store and the corresponding views?)
+    readonly property alias contactShowcaseCommunitiesModel: contactShowcaseModels.adaptedCommunitiesSourceModel
+    readonly property alias contactShowcaseAccountsModel: contactShowcaseModels.adaptedAccountsSourceModel
+    readonly property alias contactShowcaseCollectiblesModel: contactShowcaseModels.adaptedCollectiblesSourceModel
+    readonly property alias contactShowcaseSocialLinksModel: contactShowcaseModels.adaptedSocialLinksSourceModel
+
+    readonly property SQUtils.QObject adaptors: SQUtils.QObject {
+        ProfileShowcaseModelAdapter {
```

Comment:
I know that's intermediate step, but next we should move that adaptor outside the store.